tests/platform_tests/test_advanced_reboot.py

```diff
@@ -77,6 +78,18 @@ def pytest_generate_tests(metafunc):
         metafunc.parametrize("sad_case_type", sad_cases, scope="module")
 
 
+@pytest.fixture(scope="function")
+def skip_on_low_cpu_platform(request, duthosts, rand_one_dut_hostname):
+    '''
+    skip the test on some low CPU platform if the neighbor is not vsonic
+    '''
+    LOW_CPU_PLATFORMS = ["x86_64-mlnx_msn2700-r0"]
+    duthost = duthosts[rand_one_dut_hostname]
+    platform = duthost.facts["platform"]
+    if platform in LOW_CPU_PLATFORMS and request.config.getoption("neighbor_type") != "sonic":
```

Comment:
When we use vsonic as VM, the "neighbor_type" will be "sonic" right?

Reply:
Yes, that's true. Same code is used here
https://github.com/sonic-net/sonic-mgmt/blob/87a7d2185b282a72bb9f47c52deea50b09a50bc4/tests/conftest.py#L619